src/patternfly/base/patternfly-common.scss

```diff
@@ -68,9 +68,59 @@
   // stylelink-enable declaration-no-important
 }
 
+.#{$pf-prefix}m-gradient-border {
+  position: relative;
+
+  &::before {
+    position: absolute;
+    inset: 0;
+    pointer-events: none;
+    content: "";
+    background: linear-gradient(to right, #f56e6e 0%, #876fd4 65%, #5e40be 100%)
+      border-box;
+    border: var(--pf-t--global--border--width--extra-strong) solid transparent;
+    border-radius: inherit;
+    mask: linear-gradient(#000 0 0) padding-box, linear-gradient(#000 0 0);
+    mask-composite: exclude;
+  }
+}
+
+@property --#{$pf-global}--pulse--BoxShadow--Spread {
+  syntax: "<length>";
+  initial-value: 0;
+  inherits: false;
+}
+
+.#{$pf-prefix}m-pulse {
+  &,
+  &:focus,
+  &:focus-within {
```

Comment:
I don't think you need these? I believe they were in the POC initially to fire on focus, but this animation turn on/off by toggling the class.